.github/workflows/publish-pypi.yml

```diff
@@ -0,0 +1,50 @@
+name: Publish Pypi
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish:
+    name: publish
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Set up Python 2.7
```

Comment:
Python 2.7 is beyond deprecated. Here's a pypi action that uses a more recent python: https://github.com/pypa/gh-action-pypi-publish